tests/test_other.py

```diff
@@ -11175,6 +11175,11 @@ def test_unistd_mkdir(self):
   def test_unistd_cwd(self):
     self.do_run_in_out_file_test('wasmfs/wasmfs_chdir.c')
 
+  def test_unistd_unlink(self):
+    self.set_setting('WASMFS')
+    self.emcc_args = self.emcc_args.copy() + ['-DWASMFS', '-DNO_SYMLINK=1']
+    self.do_runf(test_file('unistd/unlink.c'), 'success')
```

Comment:
Could this instead be added to the existing `test_unistd_unlink`, by adding the wasmfs decorator to it?

Reply:
I think this new test addition is no longer needed?